src/backend/src/services/projects.services.ts

```diff
@@ -602,4 +602,32 @@ export default class ProjectsService {
       })
     ).map(linkTypeTransformer);
   }
+
+  /**
+   * Create a new material type
+   * @param name the name of the new material type
+   * @param submitter the user who is creating the material type
+   * @throws if the submitter is not a leader or the material type with the given name already exists
+   */
+  static async createMaterialType(name: string, submitter: User): Promise<Material_Type> {
+    if (!isLeadership(submitter.role)) throw new AccessDeniedAdminOnlyException('create material type');
```

Comment:
this should be an AccessDeniedException with a message that only leadership or above can create a material type. You can make a new AccessDeniedLeadershipOnlyException if you want